docs/recipes/correlate.qmd

```diff
@@ -0,0 +1,42 @@
+---
+title: Correlate
+execute:
+  warning: false
+---
+
+
+Here, we demonstrate how to utilize the `Correlate` module to perform cross-correlation between a specific channel/time sample and other channels/time samples within a given patch. By doing so, we create a new "correlate patch", which is essentially a data construct that encapsulates the results of these cross-correlation operations.
+We use the Ricker wavelet as an example signal and compute its cross-correlation using DASCore.
+
+
+## Load and visualize the Ricker wavelet
+
+```{python}
+import dascore as dc
+
+patch = dc.get_example_patch(
+    "ricker_moveout",
+    velocity=100,
+    duration=2,
+)
+
+patch.viz.waterfall()
```

Comment:
I suggest changing this to:

```python
patch.viz.waterfall(); 
```

so the axis string representation doesn't show up in the output.

![image](https://github.com/DASDAE/dascore/assets/11671536/07c7bf57-1173-453b-a359-160acf5855f1)